internal/cmd/frontend.go

```diff
@@ -52,15 +53,21 @@ var Frontend = &cobra.Command{
 
 		if listen != "" {
 
+			frontendTLSConfig, err := tlsConfigFromFlags(fTLSFrontend)
+			if err != nil {
+				return err
+			}
+
 			slog.Info("Starting frontend",
 				"mode", "sse",
+				"tls", frontendTLSConfig != nil,
 				"backend", backendURL,
 				"listen", listen,
 				"sse", sseEndpoint,
 				"messages", messageEndpoint,
 			)
 
-			proxy = frontend.NewSSE(listen, backendURL, tlsConfig,
+			proxy = frontend.NewSSE(listen, backendURL, backendTLSConfig, frontendTLSConfig,
```

Comment:
invert the tls config, since this is the frontend, the first tls config should be for the frontend http server